requirements.txt

```diff
@@ -64,7 +64,7 @@ python-dateutil==2.9.0
 contextlib2==21.6.0
 
 django-bootstrap3==24.2
-requirements/src/sorl-3.3-py2.py3-none-any.whl
+sorl-thumbnail==12.10.0
```

Comment:
`sorl-thumbnail` can be removed. I wasn't aware of this more modern sorl library - that's good to know about! I think we would need some more extensive work to port our ImageBlock models to this, though.